src/view/components/calculation-history/CalculationHistory.tsx

```diff
@@ -13,36 +14,65 @@ type CountryDropdownType = {
 export const CalculationHistory = ({
     calculationHistory,
 }: CountryDropdownType) => {
+    const [isListClosed, setIsListClosed] = React.useState(true);
     return (
-        <div>
-            <ul>
-                {calculationHistory.map((calculation, index) => (
-                    <li key={index}>
-                        <p>Bytes: {formatBytes(calculation.bytes)}</p>
-                        <p>
-                            Emissions: {formatEmissions(calculation.emissions)}
-                        </p>
-                        <p>
-                            Specific Emissions:{" "}
-                            {formatEmissions(calculation.specificEmissions)}
-                        </p>
-                        <p>
-                            Date & Time:{" "}
-                            {unixTimeToDateString(calculation.unixTimeMs)}
-                        </p>
-                        <p>User: {calculation.userType}</p>
-                        <ul>
-                            {Array.from(calculation.selectedCountries).map(
-                                ([countryName, percentage]) => (
-                                    <li key={countryName}>
-                                        {countryName}: {percentage}
-                                    </li>
-                                )
-                            )}
-                        </ul>
-                    </li>
-                ))}
-            </ul>
-        </div>
+        <>
+            {isListClosed ? (
+                <Button
+                    colour="light-green"
+                    type="text"
+                    onClick={() => setIsListClosed(false)}
```

Comment:
Can we make this a non-anonymous function? Like `handleViewHistoryButtonPress = () => setIsListClosed(false)`